src/scripts/main.js

```diff
@@ -2,7 +2,19 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const header = document.querySelector('.dashboard').rows[0];
+const newRow = document.createElement('tr');
```

Comment:
```suggestion
const table = document.querySelector('tbody');
```
we just need to find table and insert each row beforeend